content/en/blog/_posts/2020-12-10-Pod Impersonation and Short-lived Volumes in CSI Drivers.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: blog
+title: 'Kubernetes 1.20: Pod Impersonation and Short-lived Volumes in CSI Drivers'
+date: 2020-12-10
+slug: kubernetes-release-1.20-Pod-Impersonation-and-Short-lived-Volumes-in-CSI-Drivers
```

Comment:
```suggestion
slug: kubernetes-1.20-pod-impersonation-short-lived-volumes-in-csi
```
?

Reply:
Addressed this with the latest commit, thanks!